### PR TITLE
To do list structure

### PR DESCRIPTION
**Pull request by marurevi**
## Project improvements
- Readme update
- Linters
- Array of Objects
- Function to iterate into our array and populate an HTML list item element for each task.
- Creation of a style.css and set rules for the To Do List.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,14 +1,100 @@
...
-    element.classList.add('hello');
-  
-    return element;
-  }
-  
-  document.body.appendChild(component())
\ No newline at end of file
+import './style.css';
+
+// Lodash, now imported by this script
```
Comment (nicupop729) [inline src/index.js]: - [x] Please remove all the commented code from your files as it makes your code look unorganised, messy, and cluttered.

File: src/index.js [hunk 2]
```diff
@@ -1,14 +1,100 @@
...
+
+const list = document.getElementById('uList');
+// const entryTask = document.getElementById('newTask');
```
Comment (nicupop729) [inline src/index.js]: - [x] Please remove all the commented code from your files as it makes your code look unorganised, messy, and cluttered.

File: src/index.js [hunk 3]
```diff
@@ -1,14 +1,100 @@
...
+
+entryTask.addEventListener('keypress', (e) => {
+  if (e.key === 'Enter') {
+    savedata();
+    retrivedata();
+  }
+});
+
+window.addEventListener('load', retrivedata);
+*/
```
Comment (nicupop729) [inline src/index.js]: - [x] Please remove all the commented code from your files as it makes your code look unorganised, messy, and cluttered.

**Review by nicupop729 — CHANGES_REQUESTED** [review]
Hi @marurevi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
 - [X] Correct GitHub flow practice :heavy_check_mark:
 - [X] Good commit messages :heavy_check_mark:
 - [X] Professional README :heavy_check_mark:
 - [X] Linters are passing :heavy_check_mark:

## Required Changes ♻️

- [x] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @marurevi,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**